src/index.html

```diff
@@ -12,7 +12,48 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        class="product-card__image"
+        alt="imac"
+      />
+
+      <div class="product-card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="product-card__code">Product code: 195434</div>
+
+      <div class="stars">
+        <div class="stars-4">
+          <div class="stars-4__star"></div>
+          <div class="stars-4__star"></div>
+          <div class="stars-4__star"></div>
+          <div class="stars-4__star"></div>
+          <div class="stars-4__star"></div>
+        </div>
+        <span class="reviews">Reviews: 5</span>
+      </div>
+
+      <div class="price">
+        <span class="price__text">Price:</span>
+        <span class="price__number">$2,199</span>
+      </div>
+
+      <a
```

Comment:
It's recommended to use a button element for actions like 'buy' instead of an anchor tag. Anchor tags are intended for navigation, while buttons are meant for actions.